tests/count_words_test.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+from src import count_words
+
+
+class TemplateTest(unittest.TestCase):
+
+    @staticmethod
+    def test_gets_words_from_string():
```

Comment:
oh! now this is the fun part:
the name rule for tests is that the name should tell you the spec.
in other words, when a test fails, you want to know exactly what you wanted to check and failed to do.

in this example, the test name is `test_gets_words_from_string`. who? what's checked here? if this fails, what's broken?

Reply:
another guidline is that each test should only check one thing. in this test I already noticed 3 checks:
1. you are checking that punctuation marks are not words or part of words
2. you are checking that sequences of letter *are* words
3. you are checking that random character sequences (~~) are not considered words

Reply:
Nice!!! Alright, 
I'll have another go on this, didn't think about it that way. 
I thought unit tests are testing basically each function.